### ISSUE TITLE
[a11y] [Banners] In card and modal banners low contrast against page background

### ISSUE THREAD

**Issue by sarahill** [Shopify/polaris-react#1412]
## Banner in card / modal

Related to #1393 and #1303 

In context banners are used to display non-critical info within the content of the page without detracting from the merchant's goal.

## Problem

These banners have low contrast against backgrounds and rely on color to convey status.

- Colour shouldn’t be the only thing that portrays meaning in our banners, the icon helps but we could also have some text
- We need to look into a new way to make the banner in card more perceivable the current background provides no value as the contrast is too low and the spacing is the only thing making it appear different to the content.

## Examples

![image](https://user-images.githubusercontent.com/8629173/57239046-cf2e4780-6ff8-11e9-9bf2-e9af2f873f03.png)

**Comment by sarahill** [comment]
@dpersing Do you have any thoughts on this?

**Comment by dpersing** [comment]
I wouldn't say that we rely on color to convey on status, since we do also use icons and and text. Using color is totally cool, it just can't be the _only_ way we convey information. But, I totally agree that we could make banners stand out more, since as you mention the spacing and colors are very subtle.

Off the top of my head things:

- We could make the icon bigger
- We could make the banner border more pronounced (which would also mean we would need to look at its focus state as well, which currently uses a solid all-around border)
- We could bump up the heading font size

We should also keep in mind the explorations around stacking multiple banners (https://github.com/Shopify/polaris-ux/issues/218).

This might be a good Frideation topic to collaborate on? @sarahill

**Comment by sarahill** [comment]
> I wouldn't say that we rely on color to convey status since we do also use icons and text.

☝️ That's true. I don't love how much the banner blends into the page background but it may be a lower priority.

Frideations would be a great way to dig into this more! Happy to pair with you on it.

**Comment by dpersing** [comment]
While @sarahill and I poked around admin looking at banner examples, we also found some non-banner items that probably should be banners. We'll catalog them here to take their use into account as well.

A non-banner case where I feel like a banner would be appropriate, on the order page:

<img width="979" alt="Screenshot highlighting warnings that the merchant has not set up channels for Google or Facebook correctly, but hidden in the sidebar" src="https://user-images.githubusercontent.com/1462085/57550523-8be71800-731b-11e9-925b-ec88a70bfe7d.png">

**Comment by sarahill** [comment]
Link to explorations https://shopify.invisionapp.com/share/H5RYQ3JVN3G#/362744229_In_Context_Banner_Grouped

Devon and I explored some options to make these banners pop a bit more against the page or card background. We added a `1px` top border to carry over the style from the page banners and help these standout.

![image](https://user-images.githubusercontent.com/8629173/57551721-be9f0980-7337-11e9-826e-5714031f69e6.png)

This takes the concept a bit further and incorporates the multi-banner handling designs that Amanda and Patrick worked on.
![image](https://user-images.githubusercontent.com/8629173/57551687-adee9380-7337-11e9-88a7-17b7436926a3.png)

**Comment by shutingc** [comment]
Do we know when this will be shipped roughly?

**Comment by dpersing** [comment]
Hi @shutingc! We don't yet have a timeline for shipping this.
